coding-concepts.js

```diff
@@ -9,36 +9,36 @@
 // --------------------1) What will this log?
 
 const cohort = "Foxtrot 2023"
-// console.log(cohort.split(""))
+console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ['F', 'o', 'x', 't','r', 'o', 't', ' ', '2', '0', '2', '3'] .split("") is a string method that that seperated the string it is given into an array of stringa 
+// b) Verify and explain: .split("") is a string method that devides a string into substring and puts it orderly into an array.
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
-  `Hello, ${name}!`
+  return `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Hello, LEARN Student! This is a function called greeter where we use a string interpelation in order to alter the text within that function.
+// b) Verify and explain: it logs undefined, obviously! EVERY FUNCTION MUST HAVE RETURN :)))) Every single function must have a return. tricky, very tricky.
 
 // --------------------3) What will this log?
 
 const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2)
-// console.log(multipliedByTwo)
+console.log(multipliedByTwo)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: i am getting cocky: [8, 10, 12, 14, 16]?
+// b) Verify and explain: oooh, yes! ok array.map = is a for loop; .map(an anonymous function was used here and this is where i though the trick was). Apparently NOT. an Anonymous function works eventhough it is anonymous. 
```

Comment:
This is just written in a short version, it is the same thing as this:
`[4, 5, 6, 7, 8].map((number) => { return number * 2 })`